data/tokens.json

```diff
@@ -1027,6 +807,18 @@
       "gtmds_color-style_disabled-color": {
         "value": "rgba({gtmds_color_white}, 0.6)",
         "type": "color"
+      },
+      "gtmds_color-style_hover-color": {
+        "value": "rgba({gtmds_color_gray-9}, {gtmds_opacity-hover})",
+        "type": "color"
+      },
+      "gtmds_color-style_selected-color": {
+        "value": "rgba({gtmds_color_gray-9}, {gtmds_opacity-selected})",
+        "type": "color"
+      },
+      "gtmds_color-style_selected-hover-color": {
+        "value": "rgba({gtmds_color_gray-9}, {gtmds_opacity-selected-hover})",
+        "type": "color"
```

Comment:
I think those should be "gtmd_color-style_color-selected" , just invert the place since it will be more consistent with everywhere else we reuse states.